Cakes-Co/style.css

```diff
@@ -1 +1,400 @@
-/*  Add your styling here */
+/* Add your styling here */
+:root {
+    --bg-color: #e7e5e5;
+    --secondary-color: #f8f8f8;
+    --accent-color: #823a85;
+    --logo-size: 15vh;
+    --nav-color: #120990;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+/* design using the Mobile First approach */
+.logo-container {
+    display: grid;
+    grid-template-areas: "logo text"; 
+    align-items: center;
+    justify-content: space-between;
+    padding: 1vh 3vw;
+    margin-bottom: 1vh;
+    background-color: var(--bg-color);
+}
+
+.svg-container {
+    width: var(--logo-size);
+    height: auto;
+    grid-area: logo; 
+}
+
+.logo-text {
+    font-weight: bold;
+    grid-area: text; 
+    align-self: end;
+    font-style: italic;
+    color: var(--accent-color);
+    text-align: right; 
+    font-size: 1.9rem;
+}
+
+.welcome-container {
+    display: grid;
+    gap: 20px;
+    grid-template-areas: 
+    "welcome"
+    "cake";
+    margin: 20px;
+}
+
+.welcome-cake{
+    grid-area: cake; 
+}
+
+.welcome-cake img {
+    width: 100%;
+    height: fit-content;
+    border-radius: 5px;
+    transition: all 0.3s ease; 
+}
+
+.welcome-cake img:hover {
+    opacity: 0.8; 
+    transform: scale(0.9); 
+}
+
+       
+.welcome-text {
+    grid-area: welcome;
+    text-align: left; 
+    line-height: 1.5;
+    font-size: clamp(3vh, 2.5vw, 4vh);
```

Comment:
Interesting - can you talk through why you have mixed height and width sizing here?